api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -212,37 +211,25 @@ Random random() {
   @Captor private ArgumentCaptor<LeonardoCreateRuntimeRequest> createRuntimeRequestCaptor;
   @Captor private ArgumentCaptor<LeonardoUpdateRuntimeRequest> updateRuntimeRequestCaptor;
 
-  @MockBean LeonardoRuntimeAuditor mockLeonardoRuntimeAuditor;
```

Comment:
if we used this field: moved down and grouped with others
if we needed to mock it but didn't need the field: moved into the collective MockBean
else: removed